ksql-engine/src/main/java/io/confluent/ksql/internal/KsqlEngineMetrics.java

```diff
@@ -231,4 +253,30 @@ private Sensor createSensor(final Metrics metrics, final String sensorName) {
     return sensor;
   }
 
+
+  private Sensor configureNumActiveQueriesForGivenState(final Metrics metrics,
```

Comment:
Return `Gauge<Long>` rather than `Sensor`, which is a double. A 'count' is better represented as a long...

(Same for other count metrics in this class).

Reply:
I changed them to `Gauge<Long>` as you suggested.